yt/utilities/png_writer.py

```diff
@@ -1,21 +1,29 @@
-import matplotlib._png as _png
-
 from io import BytesIO
 
-def call_png_write_png(buffer, width, height, fileobj, dpi):
-    _png.write_png(buffer, fileobj, dpi)
+try:
+    # matplotlib switched from an internal submodule _png to using pillow (PIL)
+    # between v3.1.0 and v3.3.0
```

Comment:
This is a good solution for the imports. Thanks for adding the message too for our future devs. 🙂